src/client/cloud.rs

```diff
@@ -17,7 +17,7 @@ use tokio::runtime::Runtime;
 use tokio::{select, time::Instant};
 
 const CLOUD_URL: &str = "https://cloudapi.cloud.couchbase.com";
-pub const CAPELLA_SRV_SUFFIX: &str = "cloud.couchbase.com";
+pub const CAPELLA_CONNSTR_PREFIX: &str = "couchbases://";
```

Comment:
Change to `couchbases` prefix from the suffix otherwise this does not work with local dev